legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/testSuiteCompletenessTests.pure

```diff
@@ -7,7 +7,7 @@ function <<test.Test>> meta::relational::tests::testSuite::ensureEveryDynaFnIsTe
     meta::relational::tests::dbSpecificTests::sqlQueryTests::selectSubClauses::aggregationDynaFns
   ];
   let testedDynaFnNames = $packagesContainingDynaFnTests.children->filter(c| $c->instanceOf(Package)).name;
-  let testingIgnoredDynaFnNames = ['add', 'averageRank', 'cbrt', 'case', 'convertDate', 'convertDateTime', 'convertVarchar128', 'dayOfWeek', 'decodeBase64', 'denseRank', 'distinct', 'divide', 'encodeBase64', 'exists', 'group', 'if', 'isAlphaNumeric', 'isDistinct', 'isNumeric', 'minus', 'not', 'objectReferenceIn', 'parseDate', 'parseDecimal', 'parseFloat', 'parseInteger', 'parseJson', 'percentile', 'plus', 'rank', 'rowNumber', 'sub', 'times', 'toOne', 'toString', 'extractFromSemiStructured'];
+  let testingIgnoredDynaFnNames = ['add', 'averageRank', 'cbrt', 'case', 'convertDate', 'convertDateTime', 'convertVarchar128', 'dayOfWeek', 'decodeBase64', 'denseRank', 'distinct', 'divide', 'encodeBase64', 'exists', 'group', 'if', 'isAlphaNumeric', 'isDistinct', 'isNumeric', 'minus', 'not', 'objectReferenceIn', 'parseDate', 'parseDecimal', 'parseFloat', 'parseInteger', 'parseJson', 'percentile', 'plus', 'rank', 'reverseString', 'rowNumber', 'sub', 'times', 'toOne', 'toString', 'extractFromSemiStructured'];
```

Comment:
why we adding `reverseString` in ignored list. I believe plan was to reduce this list and add nothing else?

Reply:
there are some implementations that were not possible, we can revise at later date

Reply:
https://github.com/finos/legend-engine/blob/master/legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-execution/legend-engine-xt-relationalStore-executionPlan-connection/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/LegendH2Extensions.java#L41

As discussed example udf implementations, you will need to register these function in DefaultH2AuthenticationStrategy